bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteConfig.java

```diff
@@ -1,6 +1,6 @@
 package de.agilecoders.wicket.extensions.markup.html.bootstrap.editor;
 
-import java.util.Arrays;
+import com.google.common.collect.Lists;
```

Comment:
Actually we are trying to get rid of Guava usage.

Reply:
is this code acceptable ?

new ArrayList<>(Arrays.asList("picture", "link", "video", "table", "hr"));

Reply:
Affirmative